### ISSUE TITLE
Love the project

### ISSUE THREAD

**Issue by PatrickAlphaC** [tradeboard-io/tradeboard-io#1]
Have you considered using Chainlink Keepers to automate the portfolio rebalancing?

**Comment by mizdler** [comment]
Yes we have considered using Chainlink Keepers but there are two problems:
1- Upkeep costs fee to check balancing constantly and therefore makes index fund more expensive.
2- Auto swap of tokens need many considerations and also costs fee.

One of our goals keep costs of running an index fund or mutual fund as low possible while make it easy for people to enter and exit.
BTW we are talking with guys at Chainlink to make Tradeboard cross chain using Chainlink CCIP!

**Comment by PatrickAlphaC** [comment]
That would be SO COOOL!!

**Comment by PatrickAlphaC** [comment]
And understood. I'd imagine it would be nice to have as a backup though, as let's say your project goes months with no money coming in or out, it would never rebalance.

Just curious if you thought about it. Very cool

**Comment by mizdler** [comment]
I think If the volume of deviation from portfolio becomes meaningful then the bonus for correction would be big enough (compare to transaction fees) to incentives actors (individuals and bots) to fix the deviation and get the bonus otherwise it's not financially reasonable to fix the deviation and we better leave it little unbalanced.

**Comment by mizdler** [comment]
Because amount of bonus and penalty is relative to volume of deviation.
